geoutils/georaster.py

```diff
@@ -248,6 +248,14 @@ def from_array(cls, data, transform, crs, nodata=None):
         if len(data.shape) < 3:
             data = np.expand_dims(data, 0)
 
+        # Preserves input mask
+        if isinstance(data, np.ma.masked_array):
+            if nodata is None:
+                if np.sum(data.mask) > 0:
```

Comment:
I didn't know `np.sum()` works on boolean arrays! Makes me wonder why `np.count_nonzero()` exists, since it basically offers the same functionality..

Reply:
Yes, count_nonzero and sum will return the same value for boolean, but not for arrays containing other values than 0 and 1!

Reply:
Ah, thanks for the clarification!